### ISSUE TITLE
KIF iOS <9.0 cannot find accessibility labels

### ISSUE THREAD

**Issue by CurlyHeir** [kif-framework/KIF#884]
KIF seems to not work on device with iOS 8.0.2 and simuator 8.1, 8.2.
here is simple project that shows that issue, just download 8.1 iOS simulator to reproduce

https://github.com/CurlyHeir/ios-jenkins-test-runs/commits?author=CurlyHeir

this project is working with for instance iOS 9.2 on both - simulator and device

**Comment by phatmann** [comment]
Can you try using the HEAD branch and see if it works? Thanks.

**Comment by CurlyHeir** [comment]
i tested it on device with iOS 8.4.1 (12H321) and still no luck
KIF 3.5.1

**Comment by mikelupo** [comment]
I am using iOS 8.4.1 on a real iPhone 5C and KIF latest HEAD. 
I'll try your proj on Monday when I return to the office.

**Comment by CurlyHeir** [comment]
ok, i would be very thankful :)

**Comment by mikelupo** [comment]
@CurlyHeir Not waiting till Monday to check this out. 

Just curious, is this all on XCode 8 for you?

I ask because I just switched my end point device iPhone 5C to be driven by Xcode 8 (it was Xcode 7.3.1). As soon as I did that, the app launches, but nothing happens. 
Also, I just tried it locally on my own machine with an iOS 5 Simulator running iOS 8.4. Same outcome that see on my actual device in the CI lab.

That said, if you're on Xcode 8 currently, what if you switch back to Xcode 7.3.1?

@phatmann, I am not familiar enough with the underpinnings of KIF to hypothesize. Anything you can contribute would be of value. 

Thanks!

**Comment by CurlyHeir** [comment]
@mikelupo 
yes, i'm currently working on XCode 8, but if i remember correctly i was working on xcode 7.3.1 when i created this issue. 

I remember some issues with KIF when iOS 10 was released, something with accessibility inspector. maybe those changes broken support for iOS 8.4.1

**Comment by mikelupo** [comment]
@CurlyHeir Thanks for the quick answer.

Seems I have answered my own question.
XCode 8 + iOS 8.4 combination does not currently work.

Based off of http://www.openradar.me/28095069
.... and there are a great deal of other posts of this nature describing this problem.
KIF depends on the XCode Unit Test framework so if Unit tests won't run, then KIF can't run.

restating what I said earlier:
I have no problem with using KIF to run my tests on XCode 7.3.1 and iOS 8.4 (Sim/hardware)

**Comment by mikelupo** [comment]
@CurlyHeir please reopen this if you try out 7.3.1 and iOs 8.4 and you still have the problem. That means it's somewhere in your integration or config. I can help with that.

**Comment by CurlyHeir** [comment]
@mikelupo 
thanks for clear and solid explanation.

**Comment by mikelupo** [comment]
@CurlyHeir FWIW I just installed the XCode 8.1. beta 3. I am able to run my KIF Tests again on my iOS 8.4 device.
Fixed? Who knows. Apple didn't say in the release notes. I'll need to put some milage on this, but it seems that the tests are running.
